pkgs/development/tools/casperjs/default.nix

```diff
@@ -28,9 +28,9 @@ in stdenv.mkDerivation rec {
     make test
   '';
 
-
   installPhase = ''
-    mv $PWD $out
+    mkdir $out
```

Comment:
When the destination is on the same mount point as the source, `mv` is almost instantaneous, whereas `cp` needs to (1) actually copy the files and (2) keep the duplicates until the build finishes.

This use of `mv` looks very common actually. The only thing that might be strange is the combination with `find`, but there are precedents too.